src/scss/components/dropdown-select/index.scss

```diff
@@ -1,9 +1,5 @@
-@import (reference) 'vars';
-
-// ===== MIXINS ==========
-@import (reference) './SelectControl';
-@import (reference) './SelectFacade';
-
+@import "vars";
```

Comment:
Please remove "vars" import. It is already imported at the top of `_config.scss`.

Reply:
Changes have been done, please review.